src/button/cta.scss

```diff
@@ -8,18 +8,17 @@
 @mixin iui-button-cta {
   @include iui-button-cta-base;
 
-  &:hover,
-  &:active,
   &:focus {
-    @include themed {
-      background-color: t(iui-color-background-positive-overlay);
-      border-color: t(iui-color-background-positive-overlay);
-      color: t(iui-color-foreground-accessory);
+    @include iui-button-cta-hover-focus;
+
+    &:focus:not(:focus-visible):not(:hover) {
```

Comment:
This line should also follow the `&:where(:not(:focus-visible)) {` structure I believe.

Reply:
Fixed.